JS/Lesson34/goksel/recipe-app/src/AppRouter.jsx

```diff
@@ -0,0 +1,40 @@
+import { Navigate, Route, Routes } from "react-router-dom"
+import { NewRecipe } from "./modules/recipes/NewRecipe/NewRecipeForm"
+import { RegistrationForm } from "./modules/user/RegistrationForm";
+import { Settings } from "./modules/user/Settings";
+import { RecipeDetail } from "./modules/recipes/RecipeDetail";
+import { SearchBar } from "./modules/recipes/SearchBar";
+import { RecipeList } from "./modules/recipes/RecipeList";
+
+export const AppRouter = ({ recipes, setRecipes, setIsSignedIn, isSignedIn }) => {
+    return (
+        <Routes>
+            <Route path="user">
```

Comment:
Add a log in page. Right now registration page is displayed instead of log in page but they should be different pages.

Reply:
I added log in page.